exporters/common/src/main/java/io/opentelemetry/exporter/internal/marshal/Serializer.java

```diff
@@ -340,6 +353,25 @@ protected abstract void writeStartRepeatedVarint(ProtoFieldInfo field, int paylo
 
   protected abstract void writeEndRepeatedVarint() throws IOException;
 
+  /** Serializes a {@code repeated int32} field. */
+  public void serializeRepeatedInt32(ProtoFieldInfo field, List<Integer> values)
```

Comment:
Nit: Maybe take `Collection` instead of `List` for more flexibility here.

Reply:
`List` is already used elsewhere in methods serializing repeated fields. I agree that `Collection` is more flexible, but shouldn't obligate @jhalliday to make that switch here. If Jonathan wants to follow the "boy scout rule", and bundle the switch to Collection in with this PR, I'm open to that. Can also open a separate issue and resolve in a followup PR.